test/unit/steps/reasons-for-appealing/SendingEvidence.test.js

```diff
@@ -32,7 +32,7 @@ describe('SendingEvidence.js', () => {
         });
 
         it('should be false when the email address has not been defined', () => {
-            sendingEvidence.fields.emailAddress.value = '';
+            sendingEvidence.fields.emailAddress.value = undefined;
```

Comment:
I'm not sure if this is correct. Have you checked that one-per-page assigns `undefined` to fields that haven't been filled out my the user. I was under the impression it would always be an empty string.